test/events/events.go

```diff
@@ -37,9 +35,9 @@ func CheckEvents(t *testing.T, fr *record.FakeRecorder, testName string, wantEve
 
 // CheckCloudEvents checks that all events in wantEvents, and no others, were received by the FakeClient
 // in any order.
-func CheckCloudEvents(t *testing.T, fce *FakeClient, testName string, wantEvents []string) error {
```

Comment:
the `FakeClient` has been removed in favor of the string chan because `FakeClient` has been exported as part of the cloudevents package since forever and would cause a circular dependency if kept here. Refactoring where _that_ lives is a whole different can of worms.